content/faq/docker/docker-machine.md

```diff
@@ -50,16 +50,15 @@ docker-machine create my-awesome-ec2-project \
 --driver amazonec2 \
 ```
 
-This line allowed you to create a new instance Ubuntu machine, describing which region to settle down.
+This line allowed you to create a new instance Ubuntu machine. By the way, If you've already configured the AWS Access Key and Secret Key for using at the AWS CLI's local machine. The Docker Machine command will bring on those kinds of secret keys from your system automatically.
 
```

Comment:
Could you add more details for beginners about how we configure access key and secret key for AWS?

Reply:
Yeah, That's a good idea. So, this article. I'd describe the reader that's passing the `access key` and `secret-key` instead of feeding by the AWS configuration file on your local computer. I think It easier than the previous one.